### DISCUSSION TITLE
Reason for z-index 11 on first th child in table?

### DISCUSSION THREAD

**Discussion by lsbyerley** [saadeghi/daisyui#703] [Q&A] [answered]
Curious why there is a `z-index: 11` set for the first `<th>` child in a table. It seems to cause more issues with the default z-index for modal/dialog's than providing any real benefit

![Screen Shot 2022-04-11 at 1 24 46 PM](https://user-images.githubusercontent.com/3066258/162795700-401ff4d7-48e6-4497-a1a5-9198330c191d.png)

**Comment by saadeghi** [comment, marked as answer]
This was a quick fix.  
You're right. It might create further issues. I will remove it soon.
